frontend/app/routes.ts

```diff
@@ -50,8 +50,8 @@ function toRouteConfigEntries(routes: I18nRoute[]): RouteConfigEntry[] {
 /**
  * see: https://reactrouter.com/dev/start/framework/routing
  */
-export const routes: RouteConfigEntry[] = [
+export default [
   index('routes/language-chooser.tsx'), //
   route('/:lang/*', 'routes/catchall.tsx'),
   ...toRouteConfigEntries(i18nRoutes),
-];
+] satisfies RouteConfigEntry[];
```

Comment:
Use `RouteConfig` type.

```suggestion
] satisfies RouteConfig;
```

Reply:
My bad homie. Will fix.